dashboard/src/app/stacks/stack-details/stack.controller.ts

```diff
@@ -103,9 +122,35 @@ export class StackController {
   }
 
   /**
+   * Handle stack's tag adding.
+   *
+   * @param tag stack's tag
+   * @returns {*}
+   */
+  handleTagAdding(tag) {
```

Comment:
you may add type of argument
like tag : any (or string I'm not checked) and as there is a return value, you may add it at the end of the method

`handleTagAdding(tag : string) : string`